app/mailers/location_or_scheme_deactivation_mailer.rb

```diff
@@ -0,0 +1,38 @@
+class LocationOrSchemeDeactivationMailer < NotifyMailer
+  DEACTIVATION_TEMPLATE_ID = "8d07a8c3-a4e3-4102-8be7-4ee79e4183fd".freeze
+
+  def send_deactivation_mail(user, log_count, update_logs_url, scheme_name, postcode = nil)
+    send_email(
+      user.email,
+      DEACTIVATION_TEMPLATE_ID,
+      {
+        log_count:,
+        log_or_logs: log_count == 1 ? "log" : "logs",
```

Comment:
You can use https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize directly in the template here :)

Reply:
When you say "in the template"; this is going out to Notify rather than ERB internally. Do we have access to it actually in the Notify template (as configured in the web interface), or do you just mean we can put `log_or_logs: pluralize(log_count, "log")` here?

tl;dr do we have access to these sort of helpers in Notify templates?